src/tests/ftest/util/command_utils.py

```diff
@@ -210,7 +210,10 @@ def stop(self):
                     self._command, str(state))
                 self._process.send_signal(signal_to_send)
                 if signal_list:
-                    time.sleep(5)
+                    try:
+                        self._process.wait(timeout=5)
+                    except subprocess.TimeoutExpired:
```

Comment:
(pylint-undefined-variable) Undefined variable 'subprocess'

Reply:
With `from subprocess import TimeoutExpired` hopefully  this change will avoid the pylint error:
```suggestion
                    except TimeoutExpired:
```